README.md

```diff
@@ -1,13 +1,11 @@
-# video.js HLS Source Handler
+# video.js HLS Tech
```

Comment:
It looks like this whole file is out of date. I think that means we need a rebase on this PR.